api/SpeechGrammarList.json

```diff
@@ -20,7 +20,14 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "44",
```

Comment:
FWIW, I cannot confirm that this interface was available in Firefox 44 with that pref enabled:

<img width="671" height="644" alt="image" src="https://github.com/user-attachments/assets/aa2e7022-e561-41ce-ae8c-ffb0515e5168" />

---

According to https://bugzilla.mozilla.org/show_bug.cgi?id=1169653, it was limited to certain apps, and https://bugzilla.mozilla.org/show_bug.cgi?id=1975468 further confirms that these interfaces were effectively never available before Firefox 142 (except on privileged pages, like `about:config`).

```suggestion
            "version_added": "142",
```

(Tested in Firefox 141, and Firefox 142.)